lib/ansible/modules/cloud/azure/azure_rm_aci.py

```diff
@@ -0,0 +1,405 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_aci
+version_added: "2.4"
+short_description: Manage an Azure Container Instance (ACS).
+description:
+    - Create, update and delete an Azure Container Instance.
+
+options:
+    resource_group:
+        description:
+            - Name of a resource group where the Container Services exists or will be created.
+        required: true
+    group_name:
+        description:
+            - Name of the container Group
+        required: true
+        default: null
+    name:
+        description:
+            - Name of the container instance.
+        required: true
+        default: null
+    image:
+        description:
+            - Image to be used to create container instance.
+        required: true
+        default: null
+    os:
+        description:
+            - Operating System
+        choices:
+            - linux
+            - windows
+        default: linux
+        required: false
+    state:
+        description:
+            - Assert the state of the ACS. Use 'present' to create or update an ACS and 'absent' to delete it.
+        default: present
+        choices:
+            - absent
+            - present
+        required: false
+    location:
+        description:
+            - Valid azure location. Defaults to location of the resource group.
+        default: resource_group location
+        required: false
+    service_principal:
+        description:
+            - The service principal suboptions.
+        required: false
+        default: null
+        suboptions:
+            client_id:
+                description:
+                    - The ID for the Service Principal.
+                required: false
+            client_secret:
+                description:
+                    - The secret password associated with the service principal.
+                required: false
+
+extends_documentation_fragment:
+    - azure
+    - azure_tags
+
```

Comment:
As we talked before, we should keep parameters the same as [Azure-CLI](https://github.com/Azure/azure-cli/tree/master/src/command_modules/azure-cli-container) does. Now the ip-address/port parameters are missing. This will cause we cannot visit the ACI we created from external network.

Reply:
that's my next change, now I am reviewing azure-cli parameters their names and descriptions :-)

Reply:
now ip_address & port is included